### PR TITLE
Added the Contact Page

### PR DESCRIPTION
**Pull request by Neelyooo**
https://github.com/kriyatech-team/kriya.tech/assets/135664115/b67fdbfc-8860-46a4-8047-2da52c3a7dfa

Added the ContactPage.jsx File and also resolved the version conflicts.

### CODE REVIEW

**Comment by arpanxdutta** [comment]
Good initial work done!

Changes needed are:
1. Why is the Contact page below the home page?
2. Where are the options to contact us via mail, social media, etc?
3. Is this form submission data integrated to Google Sheets or something else?
4. The message box size should auto-adjust based on the text size/length. Why drag the border to expand/collapse the message box?
5. This Form is more for the use-cases where users want to directly write some message for us without mailing us, or they want to add some suggestions or they want semething from the website that is missing.

File: kriya-tech/src/ContactPage.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            <textarea
+              id="message"
+              name="message"
+              className="w-full border border-gray-300 rounded px-3 py-2"
+            ></textarea>
+          </div>
+          <div className="text-center">
+            <button
+              type="submit"
+              className="bg-blue-500 text-white px-4 py-2 rounded hover:bg-blue-600"
```
Comment (arpanxdutta) [inline kriya-tech/src/ContactPage.jsx:45]: Color names like text-white, border-gray-300, text-gray-600 is hardcoded. 
What if some day we want to change the color schema or color palette of the website or of this particular page?
Can you not name these colors somewhere in a color config file and use the variable names of these colors to be used in this page by importing them?

Comment (arpanxdutta) [reply kriya-tech/src/ContactPage.jsx:45]: This is applicable throughout the entire project and this PR as well. You can name these like variables like formBackgroundColor, formWidthBorderColor, formHeightBorderColor, etc.

**Review by arpanxdutta — CHANGES_REQUESTED** [review]
please resolve all comments

File: kriya-tech/package.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.4.3",
-    "postcss": "^8.4.30",
+    "postcss": "^8.4.31",
```
Comment (arpanxdutta) [inline kriya-tech/package.json:32]: Why is this change required?

File: kriya-tech/src/App.jsx [hunk 3]
```diff
@@ -14,6 +14,7 @@ const App = () => {
       <Section />
       <Cards />
       <Footer />
+      <ContactPage />
```
Comment (priyansh17) [inline kriya-tech/src/App.jsx:17]: Please add it through Navbar instead of adding directly  below footer.

We can add few interactive buttons inside footer though which redirects to Contact page.
